Fortinet/fortimail/ingest/parser.yml

```diff
@@ -0,0 +1,190 @@
+              {% if parsed_event.message.get('type') == 'event' and 'stat=' not in parsed_event.message.get('msg') and 'relay' in parse_event_with_no_stat.msg %}'relay': '{{parse_event_with_no_stat.msg.relay}}',{% endif %}
+              {% if parsed_event.message.get('type') == 'event' and 'stat=' not in parsed_event.message.get('msg') and 'msgid' in parse_event_with_no_stat.msg %}'message_id': '{{parse_event_with_no_stat.msg.msgid}}',{% endif %}
+              {% if parsed_event.message.get('type') == 'event' and 'stat=' not in parsed_event.message.get('msg') and 'STARTTLS' in parse_event_with_no_stat.msg %}'start_tls': '{{parse_event_with_no_stat.msg.STARTTLS}}',{% endif %}
+              {% if parsed_event.message.get('type') == 'event' and 'stat=' not in parsed_event.message.get('msg') and 'class' in parse_event_with_no_stat.msg %}'class': '{{parse_event_with_no_stat.msg.class}}',{% endif %}
+              {% if parsed_event.message.get('type') == 'event' and 'stat=' not in parsed_event.message.get('msg') and 'nrcpts' in parse_event_with_no_stat.msg %}'nrcpts': '{{parse_event_with_no_stat.msg.nrcpts}}',{% endif %}
+              {% if parsed_event.message.get('type') == 'event' and 'stat=' not in parsed_event.message.get('msg') and 'stat' in parse_event_with_no_stat.msg %}'stat': '{{parse_event_with_no_stat.msg.stat}}',{% endif %}
+              {% if parsed_event.message.get('type') == 'event' and 'stat=' not in parsed_event.message.get('msg') and 'verify' in parse_event_with_no_stat.msg %}'verify': '{{parse_event_with_no_stat.msg.verify}}',{% endif %}
+              {% if parsed_event.message.get('type') == 'event' and 'stat=' not in parsed_event.message.get('msg') and 'xdelay' in parse_event_with_no_stat.msg %}'xdelay': '{{parse_event_with_no_stat.msg.xdelay}}',{% endif %}
+              {% if parsed_event.message.get('type') == 'event' and 'stat=' not in parsed_event.message.get('msg') and 'mailer' in parse_event_with_no_stat.msg %}'mailer': '{{parse_event_with_no_stat.msg.mailer}}',{% endif %}
+              {% if parsed_event.message.get('type') == 'event' and 'stat=' not in parsed_event.message.get('msg') and 'bits' in parse_event_with_no_stat.msg %}'bits_sent_ratio': '{{parse_event_with_no_stat.msg.bits}}',{% endif %}
+              {% if parsed_event.message.get('type') == 'event' and 'stat=' not in parsed_event.message.get('msg') and 'dsn' in parse_event_with_no_stat.msg %}'dsn_version': '{{parse_event_with_no_stat.msg.dsn}}',{% endif %}
+              {% if parsed_event.message.get('type') == 'event' and 'stat=' not in parsed_event.message.get('msg') and 'Hostname' in parse_event_with_no_stat.msg %}'hostname_info': '{{parse_event_with_no_stat.msg.Hostname}}',{% endif %}
+              {% if parsed_event.message.get('type') == 'event' and 'stat=' not in parsed_event.message.get('msg') and 'pri' in parse_event_with_no_stat.msg %}'priority_level_msg': '{{parse_event_with_no_stat.msg.pri}}',{% endif %}
+            }]
```

Comment:
Would it be possible to use this syntax for the `action.properties` field ? https://github.com/SEKOIA-IO/intake-formats/pull/585/files#diff-6f560942a906a55b3ccc8d977768fa02843a70fc57c103325c165e89b602cff2R222-R228

It would greatly improve the readability and maintainability IMHO

What do you think @squioc ?